src/types.jl

```diff
@@ -92,3 +92,21 @@ struct FeatureTrait <: AbstractFeatureTrait end
 abstract type AbstractFeatureCollectionTrait <: AbstractTrait end
 "A FeatureCollectionTrait holds objects of `FeatureTrait` and an `extent`"
 struct FeatureCollectionTrait <: AbstractFeatureCollectionTrait end
+
+"An AbstractCRSTrait for all coordinate reference systems"
+abstract type AbstractCRSTrait end
+"An AbstractProjectedTrait for all projected coordinate reference systems"
+abstract type AbstractProjectedTrait <: AbstractCRSTrait end
+"An AbstractGeographicTrait for all geographic coordinate reference systems"
+abstract type AbstractGeographicTrait <: AbstractCRSTrait end
+"An ProjectedTrait for all projected coordinate reference systems"
+struct ProjectedTrait <: AbstractProjectedTrait end
+"An UnknownTrait for all unknown (assumed projected) coordinate reference systems"
+struct UnknownTrait <: AbstractProjectedTrait
```

Comment:
Should this inherit from AbstractCRStrait?

Reply:
If we assume projected anyway then dispatch can just be on `AbstractProjectedTrait` right?

Reply:
Yeah, I assume you implement GeometryOps with Abstractprojected and catch both.